typescript/sdk/src/gas/oracle/types.ts

```diff
@@ -1,18 +1,11 @@
 import { BigNumber } from 'ethers';
 
-import { ChainMap } from '../../types';
-
 export enum GasOracleContractType {
   StorageGasOracle = 'StorageGasOracle',
 }
+
 // Gas data to configure on a single destination chain.
-export type StorageGasOracleConfig = {
+export type DestinationOracleConfig = {
```

Comment:
no explanation for name change (either in comment or pr description)

Reply:
will revert this